interpreter/cling/include/cling/Interpreter/InterpreterCallbacks.h

```diff
@@ -161,6 +162,13 @@ namespace cling {
     /// for instance when throwing interpreter exceptions.
     virtual void PrintStackTrace() {}
 
+    ///\brief Cling creates the code complete consumer for its child interp.
+    virtual void CreateCodeCompleteConsumer(Interpreter* child) const;
```

Comment:
Yes the definition is only in InterpreterCallbacks.cpp. This was intended like this when we had the code completion as an interpreter callback, but now that is happening in the interpreter we can move it, it's true. :)